### PR TITLE
WAR-1760 support regex in snmp verify keyword

### PR DESCRIPTION
**Pull request by kaheichan**
instead of simple text matching, support regex using the re library in python

### CODE REVIEW

**Comment by kaheichan** [comment]
PR failing because of snmp file score

File: warrior/Actions/SnmpActions/common_snmp_actions.py [hunk 1]
```diff
@@ -681,7 +681,7 @@ def verify_snmp_action(self, system_name, snmp_result, mib_string=None
                     testcase_Utils.pNote("No SNMP Result Present!", 'error')
         for element in result_list:
             if mib_string:
-                if mib_string in element[0] and snmp_result in element[-1]:
+                if re.search(mib_string, element[0]) and re.search(snmp_result, element[-1]):
```
Comment (venkat1077) [inline warrior/Actions/SnmpActions/common_snmp_actions.py]: Add this regex search support in below else block as well(line no. 690).

**Comment by kaheichan** [comment]
WAR-1560 is created to improve snmp pylint score

File: warrior/Actions/SnmpActions/common_snmp_actions.py (hunk 1 above)
Comment (kaheichan) [reply warrior/Actions/SnmpActions/common_snmp_actions.py]: Edited docstring and put the regex in else block, WAR-1560 is the ticket for pylint

**Pull request merged**